src/screens/users/user-page.tsx

```diff
@@ -20,16 +20,29 @@ const renderUser = ({item}: {item: User}) => {
 };
 
 const keyExtractor = (item: {id: string}) => item.id;
+const limit = 10;
 
 export const UserPage = () => {
-  const {data} = useQuery(GET_USER);
+  const [count, setCount] = useState(0);
+  const {data, refetch} = useQuery(GET_USER, {
+    variables: {offset: count},
+  });
+
+  const handleLoadMore = () => {
```

Comment:
esse `handleLoadMore` ele não mantem a listagem anterior né? nesse caso como faria pra voltar na listagem?

Eu acho que vc queria implementar algo parecido com infinite scroll, se for isso acho que falta essa parte de manter os dados antigos e adicionar os novos. Eu n lembro se o apollo mantem a listagem ou se tem que fazer na mão, mas tem um método que é recomendado para [paginação no apollo](https://www.apollographql.com/docs/react/data/queries/#fetchmore)

Reply:
Implementei essa paginação que vc recomendou. Agora está mantendo os dados antigos 😀👍🏻